libnopegl/src/node_media.c

```diff
@@ -272,6 +275,7 @@ static int media_prefetch(struct ngl_node *node)
 {
     struct media_priv *s = node->priv_data;
     nmd_start(s->player);
+    s->prefetched = 1;
```

Comment:
Not related to this PR but can we consider using bool ?

Reply:
do you mean `_Bool`?

Reply:
I meant `bool` as in `stdbool.h` (which is just a macro of _Bool) and it's just for internal use.

Reply:
probably ok

Reply:
But why would you not use bool at all ?